src/sentry/integrations/discord/actions/form.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+from typing import Any
+
+from django import forms
+from django.core.exceptions import ValidationError
+from django.forms.fields import ChoiceField
+
+from sentry.integrations.discord.utils.channel import validate_channel_id
+from sentry.services.hybrid_cloud.integration import integration_service
+
```

Comment:
This is one of the big pieces of new code. This is the form that shows in the create alert menu.